opentelemetry-sdk/tests/logs/test_export.py

```diff
@@ -198,8 +198,8 @@ def test_simple_log_record_processor_different_msg_types(self):
                 "Temperature hits high 420 C in Hyderabad",
                 "CRITICAL",
             ),
-            (["list", "of", "strings"], "WARN"),
-            ({"key": "value"}, "ERROR"),
+            ("['list', 'of', 'strings']", "WARN"),
+            ("{'key': 'value'}", "ERROR"),
```

Comment:
While I'm not familiar with the expected behavior for logging a collection, a type change like this is a big deal. Could you explain why this is right?

Reply:
I think this is starting to make sense to me. Is it always made a string because the formatters always produce a string?